Marlin/ultralcd_impl_DOGM.h

```diff
@@ -591,6 +591,29 @@ static void lcd_implementation_status_screen() {
     u8g.setPrintPos(12, 50);
     lcd_print(itostr3(feedrate_percentage));
     u8g.print('%');
+
+    //
+    // Filament sensor if SD is disabled
+    //
+
+    #if DISABLED(SDSUPPORT) && ENABLED(FILAMENT_LCD_DISPLAY)
+      // Show measured value
+      lcd_setFont(FONT_MENU);
+      u8g.setPrintPos(47, 50);
+      lcd_print('\xf8'); // TODO: LCD_STR_FILAMENTSIZE
+      lcd_setFont(FONT_STATUSMENU);
+      u8g.setPrintPos(56, 50);
+      lcd_print(ftostr12ns(filament_width_meas));
+
+      // Show calculated multiplier
+      lcd_setFont(FONT_MENU);
+      u8g.setPrintPos(93, 50);
+      lcd_print("\xa4"); // TODO: LCD_STR_FILAMENTMULTIPLIER
+      lcd_setFont(FONT_STATUSMENU);
+      u8g.setPrintPos(102, 50);
+      lcd_print(itostr3(100.0 * volumetric_multiplier[FILAMENT_SENSOR_EXTRUDER_NUM]));
+      u8g.print('%');
+    #endif
```

Comment:
This block can be made slightly more efficient by dropping three of the `lcd_setFont` calls. (We know the font at the start will be `FONT_STATUSMENU`.)
```cpp
#if DISABLED(SDSUPPORT) && ENABLED(FILAMENT_LCD_DISPLAY)
  // Show measured value
  u8g.setPrintPos(56, 50);
  lcd_print(ftostr12ns(filament_width_meas));
  // Show calculated multiplier
  u8g.setPrintPos(102, 50);
  lcd_print(itostr3(100.0 * volumetric_multiplier[FILAMENT_SENSOR_EXTRUDER_NUM]));
  u8g.print('%');
  lcd_setFont(FONT_MENU);
  u8g.setPrintPos(47, 50);
  lcd_print('\xf8'); // TODO: LCD_STR_FILAMENTSIZE
  u8g.setPrintPos(93, 50);
  lcd_print("\xa4"); // TODO: LCD_STR_FILAMENTMULTIPLIER
#endif
```